lib/src/model/package_graph.dart

```diff
@@ -202,7 +203,10 @@ class PackageGraph {
       allInheritableElements = {};
 
   /// A mapping of the list of classes which implement each class.
-  final Map<Class, List<Class>> _implementors = {};
+  final Map<Class, List<Class>> _implementors = LinkedHashMap(
+      equals: (Class a, Class b) =>
+          a.hrefAllowingNonCanonical == b.hrefAllowingNonCanonical,
```

Comment:
We need to take into account definingLibrary here.  Otherwise, we're likely to duplicate classes that are reexported through multiple private libraries.

Reply:
Unless I'm missing something and this is covered by other means.

Reply:
Ah ha, using ModelElement.from instead.